docs/get-started/installation.md

```diff
@@ -1,80 +1,83 @@
 ---
 title: ".NET MAUI installation"
 description: "Installation instructions for .NET MAUI."
-ms.date: 09/16/2021
+ms.date: 09/21/2021
 ---
 
 # Installation
 
-To create .NET Multi-platform App UI (.NET MAUI) apps, you need the latest preview versions of both Visual Studio 2022 and .NET 6.
+To create .NET Multi-platform App UI (.NET MAUI) apps, you need the latest preview versions of Visual Studio 2022 and .NET 6. Visual Studio 2022 should install .NET 6 for you, but if it's missing, download it from the [Download .NET 6.0 site](https://dotnet.microsoft.com/download/dotnet/6.0).
 
 [!INCLUDE [docs under construction](~/includes/preview-note.md)]
 
-Visual Studio 2022 should install .NET 6 for you, but if it's missing, download it from the [Download .NET 6.0 site](https://dotnet.microsoft.com/download/dotnet/6.0).
+## Visual Studio
 
-> [!NOTE]
-> If you previously used these instructions to install .NET MAUI, note the absence of the `maui-check` tool. This tool is no longer required to get .NET MAUI working in Visual Studio.
```

Comment:
I added this in because people were asking about it in issues, I figured we could leave it in for a release or two then remove it?